docs/main/docs/collator/testnet/testnet-collator.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 # Run a t0rn Testnet Collator
 
 
-This guide outlines the essential minimum of steps required to run a collator for t0rn - a release candidate of t3rn on the Rococo testnet. This guide uses the `v1.15.1-rc.0` release, however always use the latest available version.
+This guide outlines the essential minimum of steps required to run a collator for t0rn - a release candidate of t3rn on the Rococo testnet. This guide uses the `v1.16.1-rc.0` release, however always use the latest available version.
```

Comment:
Would separate the collator version updates in our docs from this PR as its already out of date again. This should ideally be handled as part of the collator release.

Reply:
this is merge conflict. those changes are automated with cargo release during merge to development